src/index.js

```diff
@@ -70,3 +105,51 @@ plusIcon.addEventListener('click', () => {
     input.focus();
   }
 });
+
+// select entire list
+let list = document.querySelector('.todo-list');
+
+// Add a click event listener to the list and its children
+list.addEventListener('click', (e) => {
+  if (e.target.classList.contains('js-tick')) {
+    const itemKey = e.target.parentElement.dataset.key;
+    toggleDone(itemKey);
+  }
+});
+
+// edit task descriptions
+
+list.addEventListener('click', (e) => {
+  if (e.target.tagName === 'SPAN') {
+    /* eslint-disable no-undef */
+    const icon = e.target;
+    /* eslint-disable no-undef */
+    const li = icon.parentElement;
+    list = li.parentElement;
+  }
+  if (icon.className === 'fa-edit') {
+    const span = document.querySelector('.todo-desc');
+    const input = document.createElement('input');
+    input.type = 'text';
+    input.value = span.textContent;
+    li.insertBefore(input, span);
+  }
+});
```

Comment:
- [ ] When editing a task and reloading the page the edited task should be saved in local storage right now it doesn't work  so please make sure when editing a task it should be saved

Reply:
@youmari, 
Okay. Thanks. I'll work on rectifying this as soon as possible.